.github/workflows/ci.yml

```diff
@@ -10,38 +10,43 @@ jobs:
     name: Test
     runs-on: ubuntu-latest
 
-    strategy:
-      fail-fast: false
-
     steps:
       - uses: actions/checkout@master
+      - id: install
+        uses: actions-rs/toolchain@v1
         with:
-          fetch-depth: 2
-
-      - name: Install stable Rust
-        run: rustup update stable && rustup default stable
-
-      - name: Set rustc version
-        run: echo "::set-env name=CURRENT_RUSTC_VERSION::$(rustc -V)"
+          toolchain: stable
+          override: true
+      - uses: actions-rs/cargo@v1
+        with:
+          command: install
+          args: cargo-sweep
 
-      # Caching
-      - name: Cache Cargo directories
-        uses: actions/cache@v2
+      - name: Cache directories
+        uses: actions/cache@v1
         with:
           path: |
+            ~/.cargo/registry
```

Comment:
This will slowly build up dependencies since it's not swept, though probably doesn't matter much given how small sources are.

Reply:
We could run [`cargo-cache`](https://github.com/matthiaskrgr/cargo-cache) to clean it up